src/syft/grid/duet/__init__.py

```diff
@@ -3,18 +3,20 @@
 import os
 from pathlib import Path
 import sys
-import tempfile
-import time
 from typing import Any
 from typing import Generator
 from typing import Optional
+from typing import Callable
 
 # third party
 import nest_asyncio
 import requests
 
 # syft relative
+from . import loopback_exchangeIds
+from . import manual_exchangeIds
 from ...core.node.domain.domain import Domain
+from .manual_exchangeIds import bcolors
```

Comment:
I think we should keep this 1 level above in its own file called `bcolors` and possibly refactor the "♫♫♫ text" into a separate set of strings so it's easier to change or translate and the code is cleaner.

Reply:
- bcolors: resolved in a new commit
- refactor the "♫♫♫ text": TODO